docs/source/use_cases/index.rst

```diff
@@ -81,6 +81,16 @@ For example, ``gangreen`` is a registered farmer ``tomato`` asset creator, he se
 
 We simplified asset creation to just a single command ``CreateAsset`` without the need to create complex smart contracts. One the major advantages of Hyperledger Iroha is in its ease, that allows developers to focus on the provided value of their applications. 
 
+Fund management
+--------------
+
+With the support of multisignature transaction it is possible to maintain a fund with multiple number of managers. In that scheme investment can only be done after the confirmation of the quorum participants.
```

Comment:
With the support of multisignature transaction**s** **>,<**

Reply:
it is possible to maintain a fund **by many** managers.

Reply:
In that scheme **>,<**

Reply:
investment can only be **made** after the confirmation of the quorum participants.